src/main/java/gross/gameoflife/gui/GridComponent.java

```diff
@@ -0,0 +1,88 @@
+package gross.gameoflife.gui;
+
+import gross.gameoflife.Grid;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseMotionListener;
+
+public class GridComponent extends JComponent {
+
+    private static Grid gameGrid;
```

Comment:
Do not create static variables.

I fail students for creating static variables.
You are only allowed to use static when creating `final static` constants and when making a `main()`